databallpy/match.py

```diff
@@ -830,3 +865,17 @@ def get_open_match(provider: str = "metrica", verbose: bool = True) -> Match:
         away_players=metadata.away_players,
     )
     return match
+
+
+def get_matching_full_name(full_name: str, options: list) -> str:
```

Comment:
Add docstring

Reply:
Might almost need the needleman-wunch algorithm here to find the best fit between to strings...

Reply:
I am not sure if needleman wunch is suited for comparing different fits. We will have to look into that, I propose to make it a seperate issue

Reply:
I changed this to using python's difflib.SequenceMatcher